Formula/msc-generator.rb

```diff
@@ -33,8 +34,10 @@ class MscGenerator < Formula
   end
 
   def install
-    system "./configure", *std_configure_args, "--disable-font-checks"
-    system "make", "-C", "src", "install"
+    extra_args = []
+    extra_args << "OBJCXX=/usr/bin/clang++" if OS.mac?
```

Comment:
> @carlocab This needs a (Ruby) comment at least.

I agree -- my previous suggestion was more "try this; this might work" rather than "this is code that definitely deserves to be committed".

> It's this only used for Objective C++ and not other Objective C compilation?

Yes, that's certainly how this works out. Though I don't know if there is any Objective C compilation going on. It's possible that there isn't any, and we'd also need `clang` for that if there were.

> I'm not opposed to this specific shim bypass if so but it'd be nicer to handle in Homebrew/brew eventually if Objective C++ will never work otherwise.

I think this is definitely something `brew` should know to handle, but I think we ought to evaluate to what extent `gcc` does not work for Objective C/C++ code.

Reply:
> I agree -- my previous suggestion was more "try this; this might work" rather than "this is code that definitely deserves to be committed".

Gotcha 👍🏻 



> I think this is definitely something `brew` should know to handle, but I think we ought to evaluate to what extent `gcc` does not work for Objective C/C++ code.

👍🏻

Reply:
We confirm that there are no Objective-C sources in this project, only the above mentioned Objective-C++ file (clipboard.mm).